.github/pull_request_template.md

```diff
@@ -1,28 +1,47 @@
-## Brief Information
+<!-- 
+🚨NOTE🚨 Please complete the following tasks before submitting a pull request:
 
-This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):
+- [ ] Read the CONTRIBUTING doc. see: https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md
+- [ ] Sign the CLA. see: https://cla-assistant.io/hypertrons/hypertrons-crx
+-->
```

Comment:
I think these are true tasks. [These awesome pr templates](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) use checkboxes in this way. So personally I prefer making them as checkboxes rather comments.